bitnami/mongodb/8.0/debian-12/rootfs/opt/bitnami/mongodb/templates/mongodb.conf.tpl

```diff
@@ -25,7 +25,7 @@ net:
     enabled: true
     pathPrefix: {{MONGODB_TMP_DIR}}
   ipv6: {{MONGODB_DEFAULT_ENABLE_IPV6}}
-  bindIpAll: false
+  bindIpAll: {{ALLOW_REMOTE_CONNECTION}}
```

Comment:
This is a default setting, hence any env. var modifying this behaviour should be prefixed with `MONGODB_DEFAULT_`.

That said, please note this won't allow you to edit the value for `bindIpAll` dynamically on runtime. This template is only used when building the container image, see:

- https://github.com/bitnami/containers/blob/4fe538c531073079670e646b7ef9ddabded23d15/bitnami/mongodb/8.0/debian-12/rootfs/opt/bitnami/scripts/mongodb/postunpack.sh#L25

In order to customize the configuration file at runtime, we recommend you to follow the instruction below instead:

- https://github.com/bitnami/containers/tree/main/bitnami/mongodb#configuration-file

Reply:
I understand what you mean, but in some cases, we really need a quick way to create a MongoDB instance that can be accessed remotely, even though it won't be changed during runtime. It seems that bitnami/postgres follows the same logic.

Reply:
I don't think that's a reason to configure MongoDB to bind **by default** all IPv4 and IPv6 addresses. I understand it's sth useful for your use case but it's not a secure setting that should be used except for certain development purposes.

That's why, I see to options:

- Adapt this PR so this setting is adapted during the container initialization (instead of doing so at build time).
- Use a custom configuration following the instructions I shared before.